docs-src/app-dev-context/child-workflows.md

```diff
@@ -13,3 +13,7 @@ When using a Child Workflow API, Child Workflow related Events ([StartChildWorkf
 
```

Comment:
logged in the parent Workflow ..

Reply:
(see hidden text above)

Reply:
+1 to that clarification (logged to the Parent Workflow Execution)